### ISSUE TITLE
Rename .env.development to .env

### ISSUE THREAD

**Issue by NiallJoeMaher** [codu-code/codu#99]
## Context

Due to an error on running `npx prisma db push` we are unable to use `.env.development`.
Prisma expects a `.env` not a `.env.development`.

Note, if someone has a solution that does not require renaming I will also look at that. 

## Expected Behavior

When running `npx prisma db push` I expect it to run as suggested in the readme docs. 

## Current Behavior

Currently `npx prisma db push` errors because there is no `.env` file.

**Comment by brsc2909** [comment]
@NiallJoeMaher  looks to be straignt forward enough to use multiple .env files.

if you have the dotenv-cli installed
```bash
npm install -g dotenv-cli
```
Then just
```bash
dotenv -e .env.development -- npx prisma db push
``` 
See. [https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files](https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files)

**Comment by NiallJoeMaher** [comment]
@brsc2909 I guess I can just add it to the `package.json` as a little script too which would mean people wouldn't even need to know how it works anyway. 🤔

**Comment by NiallJoeMaher** [comment]
Closed with #102 but will return to this with a different issue to get the different `.envs` working as suggested by @brsc2909
